Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/BackgroundWorkHelper.kt

```diff
@@ -14,18 +14,22 @@ import kotlin.random.Random
 object BackgroundWorkHelper {
 
     /**
-     * Calculate the time for diagnosis key retrieval periodic work
+     * Calculate the time for pcr diagnosis key retrieval periodic work
      *
      * @return Long
      *
      * @see BackgroundConstants.MINUTES_IN_DAY
      */
-    fun getDiagnosisTestResultRetrievalPeriodicWorkTimeInterval(): Long =
+    fun getPcrTestResultRetrievalPeriodicWorkTimeInterval(): Long =
```

Comment:
Is it only used by your new scheduler? Could we move it there?
I'd love to get rid of the "collect all values in one class" classes :grin:

Reply:
done!